src/OpenTelemetry.Exporter.OpenTelemetryProtocol/OtlpExporterOptions.cs

```diff
@@ -195,5 +203,18 @@ internal static void RegisterOtlpExporterOptionsFactory(IServiceCollection servi
                     configuration,
                     sp.GetRequiredService<IOptionsMonitor<BatchExportActivityProcessorOptions>>().Get(name)));
         }
+
+        private static Version GetAssemblyVersion()
+        {
+            try
+            {
+                var assemblyName = typeof(OtlpExporterOptions).Assembly.GetName();
+                return assemblyName.Version;
```

Comment:
IMHO it would be better to use either the assembly file version (blue) or even better the assembly informational version (green) instead of the assembly version (red) as it's much more granular:

![image](https://user-images.githubusercontent.com/1439341/216284470-bd86e53a-f322-4b58-9e63-e752bd8c7bcf.png)

I'd be happy to submit a PR to do that if people agree.

Reply:
+1 👍

Reply:
Opened #4143.